lib/common/global_header/global_header.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+
+import '../../constants/common_constants.dart';
+import '../../constants/global_header/global_header_constants.dart';
+import '../format_text.dart';
+
+class GlobalHeader extends StatelessWidget implements PreferredSizeWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      child: addAppBarTextWidget(), 
+      height: preferredSize.height,
+      padding: EdgeInsets.only(left: 25.0, right: 15.0),
+      color: Colors.transparent
+    );
+  }
+
+  @override
+  Size get preferredSize => const Size.fromHeight(50.0);
+}
+
+Row addAppBarTextWidget() {
```

Comment:
A suggestion - it might look better if we add the logo beside the "aspire"... what do you think?

Reply:
I agree. A small logo in the middle on under the aspire letters